Template/Tag/FacebookPixelTag.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\TagManager\Template\Tag;
+
+use Piwik\Settings\FieldConfig;
+use Piwik\Plugins\TagManager\Template\Tag\BaseTag;
+use Piwik\Validators\NotEmpty;
+
+class FacebookPixelTag extends BaseTag
+{
+
+    Public function getName()
+    {
+        return 'Facebook Pixel';
+    }
+
+    public function getDescription()
+    {
+        return 'The Facebook Pixel is a web analytics and avertising service offered by Facebook.';
+    }
+
+    public function getHelp()
+    {
+        return 'This tag lets you track website pageviews in your Facebook Ads account. To obtain the  Pixel ID please log in to your Facebook Ads account.';
+    }
+
+    public function getCategory()
+    {
+        return self::CATEGORY_ANALYTICS;
+    }
+
+    public function getIcon()
+    {
+        return 'plugins/TagManager/images/icons/F_icon.svg';
+    }
+
+    public function getParameters()
+    {
+        return array(
+            $this->makeSetting('pixelId',, FieldConfig::TYPE_STRING, function (FieldConfig $field) {
```

Comment:
fyi: there are 2 commas... maybe you meant to use `,'',` to have an empty string as default value? If you want to request a number, you could set `FieldConfig::TYPE_STRING` to an integer `FieldConfig::TYPE_INT`

Reply:
Re-added quotes, this was based on a prior suggestion to remove but seems they are needed to validate. Also updated to TYPE_INT